src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -12,7 +12,7 @@
     <Compile Include="CSharpArgumentInfoTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\src\Microsoft.CSharp.csproj">
+    <ProjectReference Include="..\pkg\Microsoft.CSharp.pkgproj">
```

Comment:
Currently this is not working today in Visual Studio, which is unable to read pkgprojs.

I believe that master today also has issues with building in Visual Studio. I tried with an upgraded version of the Nuget extensions and still got build errors.